app/src/main/java/com/duckduckgo/app/tabs/ui/GridViewColumnCalculator.kt

```diff
@@ -18,6 +18,7 @@ package com.duckduckgo.app.tabs.ui
 
 import android.content.Context
 import com.duckduckgo.app.global.view.toDp
+import com.duckduckgo.app.global.view.toPx
 import kotlin.math.min
 
 class GridViewColumnCalculator(val context: Context) {
```

Comment:
seems like you don't need the `Context` but `context.resources.displayMetric` would it be better to replace the constructor?